src/hooks/useOnLongPress/index.js

```diff
@@ -0,0 +1,55 @@
+import { useRef, useState } from 'react'
+
+import { useBreakpoints } from 'cozy-ui/transpiled/react/providers/Breakpoints'
+
+import { makeDesktopHandlers, makeMobileHandlers } from './helpers'
+
+import { useSelectionContext } from '@/modules/selection/SelectionProvider'
+
+export const useLongPress = ({
+  file,
+  disabled,
+  isRenaming,
+  openLink,
+  toggle
+}) => {
+  const timerId = useRef()
+  const isLongPress = useRef(false)
+  const [lastClickTime, setLastClickTime] = useState(0)
+  const { isDesktop } = useBreakpoints()
+  const {
+    setSelectedItems,
+    setFocusedIndex,
+    setLastSelectedIndex,
```

Comment:
`setLastSelectedIndex` is not exported by the selection provider

Reply:
good catch I forgott to save/commit the modification